suite/tests/api/ir_aarch64.h

```diff
@@ -127,6 +127,15 @@ test_instr_encoding(void *dc, uint opcode, instr_t *instr, const char *expected)
     return result;
 }
 
+const reg_id_t Xn_six_offset_0[6] = { DR_REG_X0,  DR_REG_X5,  DR_REG_X10,
+                                      DR_REG_X15, DR_REG_X20, DR_REG_X30 };
+const reg_id_t Xn_six_offset_1[6] = { DR_REG_X0,  DR_REG_X6,  DR_REG_X11,
```

Comment:
How do we decide which regs go in each of these Xn_six_offset_* arrays? Do we intend for them to cover all DR_REG_Xn (because they don't currently)?

Reply:
These are not exhaustive register sets. They're a selection based on our experience with test coverage.
More will be added later to improve coverage.